otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -30,7 +31,10 @@ data:
     nodeexporter = ""
     windowsexporter = ""
     windowskubeproxy = ""
+    podannotations = ""
```

Comment:
may be we should have just 1 setting for podannotation (called podannotation namespaces -- outside targets list). If its empty pod annotation is turned off . if it is not empty , podannotation is true for the specified namespaces ? A thought, as i feel this is a bit more confusing now with the new 'namespace keep list' pivot to the configmap.